Engine/source/gfx/gl/tGL/tGL.h

```diff
@@ -24,7 +24,8 @@
 #define T_GL_H
 #include "GL/glew.h"
 
-#define gglHasExtension(EXTENSION) GLEW_##EXTENSION
+// Slower but reliably detects extensions
+#define gglHasExtension(EXTENSION) glewGetExtension("GL_" # EXTENSION)
```

Comment:
Microsoft's "driver stub" is OpenGL 1.4 (Win XP and below is 1.1) and converts calls to D3D (or software) IIRC